backend/project.clj

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
can you please remove this change?

Reply:
Addressed [here|https://github.com/akvo/unep-gpml/commit/2843790c7ed570ec976ab5e3f9e92eb65779b6df#diff-55baed1d837e794d64536d1f19c52102be6963761097f7577b82ad0891d6827cL10]